### PR TITLE
migration: multiple values in created_by field

### LINKED ISSUE #303: 916__w: multiple subfields
in many records, in the field `916__` we have multiple occurrences of the subfield `w`. (it is the creation date in format YYYYWW, where WW is the week number).

For this case, the migration rule should pick up the **oldest** date from all the occurrences as the creation date and ignore the other

### PR DESCRIPTION
**Pull request by kprzerwa**
* closes #303

### CODE REVIEW

**Review by jrcastro2 — APPROVED** [review]

File: cds_ils/importer/providers/cds/rules/base.py [hunk 1]
```diff
@@ -72,7 +72,16 @@ def created(self, key, value):
                 }
             )
             self["created_by"] = _created_by
-            date = clean_val("w", value, int, regex_format=r"\d{6}$")
+
+            date = 99999
```
Comment (ntarocco) [inline cds_ils/importer/providers/cds/rules/base.py:76]: date `99999`? What kind of date is that???? :)

**Review by KonstantinaStoikou — APPROVED** [review]

File: cds_ils/importer/providers/cds/rules/base.py (hunk 1 above)
Comment (kprzerwa) [reply cds_ils/importer/providers/cds/rules/base.py:76]: :) this is a dirty hack, I didn't have a better idea there. The problem is that in this field we have date in format `YYYYWW` where `WW` is week number, and there might be two subfields `w`. We are supposed to choose the oldest date, so the initial 999999 is for comparing reasons. I thought it is not worth it to spend time for more sophisticated solutions.

Comment (ntarocco) [reply cds_ils/importer/providers/cds/rules/base.py:76]: Ok, makes sense, I would just add a comment (even another PR) to explain this... if you read this code in one year I would wonder what happened here... :)

**Review by ntarocco — APPROVED** [review]

**Comment by ntarocco** [comment]
It requires rebase and fix conflicts

**Pull request merged**